-------------------------------------------------------------------------------
Test set: org.alvaroaleman.factorial.FactorialTest
-------------------------------------------------------------------------------
Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.087 s <<< FAILURE! - in org.alvaroaleman.factorial.FactorialTest
org.alvaroaleman.factorial.FactorialTest.shouldReturnExceptionWhenComputeNegative  Time elapsed: 0.009 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected java.lang.RuntimeException to be thrown, but nothing was thrown.
	at org.alvaroaleman.factorial.FactorialTest.shouldReturnExceptionWhenComputeNegative(FactorialTest.java:56)

